app/rest/dbroutes.ts

```diff
@@ -139,22 +139,31 @@ export function dbroutes(router: Router, platform: Platform) {
 				req.query.from as string,
 				req.query.to as string
 			);
+			const { page, size } = req.query;
 			if (isNaN(txid)) {
 				txid = 0;
 			}
 			if (channel_genesis_hash) {
 				const extReq = (req as unknown) as ExtRequest;
-				dbCrudService
+				let data = await dbCrudService
 					.getTxList(
 						extReq.network,
 						channel_genesis_hash,
 						blockNum,
 						txid,
 						from,
 						to,
-						orgs
+						orgs,
+						page,
+						size
```

Comment:
[nitpick]Pass the default values for page and size.

Reply:
This has been handled in the UI: reference: `/api/txList/${channel}/0/0?${query?query:''}&page=${pageParams?.page||1}&size=${pageParams?.size||10}`.